### PR TITLE
Backport changes to v1.24

### PR DESCRIPTION
**Pull request by jmazzitelli**
This backports the following commits to v1.24 branch:

* https://github.com/kiali/kiali-operator/commit/8e5d04d8485284376e9e66fcd51d0be46b4dc06e
* https://github.com/kiali/kiali-operator/commit/ef176e2ac209d6aba0403d06bce5c7641d6826ff
* https://github.com/kiali/kiali-operator/commit/1d0c4c67c278c035adf38a8f294d738fc278f735
* https://github.com/kiali/kiali-operator/commit/6ad8ada2026c477f4a714314a399e09e4ede6897
* https://github.com/kiali/kiali-operator/commit/cff4c37fe29c3997a897e0a969ceb27b3236570c

### CODE REVIEW

**Comment by jmazzitelli** [comment]
We most likely want this, too - so do not merge this backport PR until this PR is merged and we backport it also: https://github.com/kiali/kiali-operator/pull/284

**Comment by jmazzitelli** [comment]
@jshaughn This should be ready for review and merge

**Review by jshaughn — APPROVED** [review]
OK, I assume the 1.33.0 files are here just because it's part of the backport, and they are not actually used but also don't hurt anything.  It's also fine to see the 1.0 files go away.  So, LOK.

**Pull request merged**
